index.js

```diff
@@ -308,8 +308,10 @@ module.exports = function StateProvider(makeRenderer, rootElement, stateRouterOp
 			parameters = extend(lastStateStartedActivating.get().parameters, parameters)
 		}
 
-		prototypalStateHolder.guaranteeAllStatesExist(stateName)
-		var route = prototypalStateHolder.buildFullStateRoute(stateName)
+		var destinationState = stateName === null ? lastStateStartedActivating.get().name : stateName
```

Comment:
Good question.  We should probably have a test for that, and throw an error with a useful message?